app/views/tweets/index.html.erb

```diff
@@ -1,27 +1,9 @@
-<p id="notice"><%= notice %></p>
-
-<h1>Tweets</h1>
-
-<table>
-  <thead>
-    <tr>
-      <th>Tweet</th>
-      <th colspan="3"></th>
-    </tr>
-  </thead>
-
-  <tbody>
-    <% @tweets.each do |tweet| %>
-      <tr>
-        <td><%= tweet.tweet %></td>
-        <td><%= link_to 'Show', tweet %></td>
-        <td><%= link_to 'Edit', edit_tweet_path(tweet) %></td>
-        <td><%= link_to 'Destroy', tweet, method: :delete, data: { confirm: 'Are you sure?' } %></td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
-
-<br>
-
-<%= link_to 'New Tweet', new_tweet_path %>
+<section class="section">
+    <div class="container">
+        <div class="columns">
+           <%= render 'feed' %>
```

Comment:
it might be good to pass the `@tweets` into the feed partial, that way you could use the partial with any list of tweets you want to use (say a feed of tweets by just one user)

Reply:
Added the `@tweets` into the feed partial.